lib/model/store.dart

```diff
@@ -226,7 +228,10 @@ class PerAccountStore extends ChangeNotifier {
   }
 
   void handleEvent(Event event) {
-    if (event is HeartbeatEvent) {
+    if (event is RealmEmojiUpdateEvent) {
+      assert(debugLog("server event: realm_emoji/update"));
+      realmEmoji..clear()..addAll(event.realmEmoji);
```

Comment:
Since we already have the new set of emoji in the form of a map, we might as well use that map rather than copy it into the existing one: `realmEmoji = event.realmEmoji`.